Packages/Tracking/Examples~/XR Examples/Example Assets/Misc and Shared/Scripts/SimpleInteractionGlow.cs

```diff
@@ -39,14 +39,16 @@ public class SimpleInteractionGlow : MonoBehaviour, IPhysicalHandHover, IPhysica
         [Tooltip("This color only applies if the object is a PhysicalHandsButton or PhyaicalHandsSlider.")]
         public Color pressedColor = Color.white;
 
-        private Material[] _materials;
+        private List<Material> _materials;
 
         [SerializeField]
         private PhysicalHandsButton _physHandButton;
 
         [SerializeField]
         private Rend[] rends;
 
+        private List<SpriteRenderer> spriteRends;
```

Comment:
Is a specific collection for SpriteRenderers necessary?

We used the base Renderer rather than MeshRenderer to allow referencing other Renderer types

Reply:
yeah - sprite renderers expect you to modify the colour component on the renderer rather than the material. I can get around this by setting all my sprites to white in the inspector, but then im unable to easily preview what my UI is going to look like until I hit play

Reply:
That's annoying! Well I'll leave it for a proper review from Bryce then :D